DATM_FORCING/DATM_Sanitycheck/ch_datm.jnl

```diff
@@ -0,0 +1,49 @@
+ can data/all
```

Comment:
Hummm ... Not too exited about ferret when we are all moving to python. 
There are no comments and everything is hard coded. Let's try to make it slightly "more right" than the last PR and try to clean up that code. Could you "pythonify" what your scripts?